binder/environment.yml

```diff
@@ -2,34 +2,34 @@ channels:
 - conda-forge
 dependencies:
 # Jupyter
-- jupyterlab=2
-- nodejs=11
-- nbconvert=5.5
-- widgetsnbextension=3.5
```

Comment:
If we keep this line, I think we'll still be able to open a widgets notebook in classic notebook from the jlab menu.

Reply:
Oh wait, never mind - it should be pulled in by ipywidgets.